development/compiling/compiling_with_mono.rst

```diff
@@ -8,7 +8,7 @@ Compiling with Mono
 Requirements
 ------------
 
-- Mono 5.12.0 or greater
+- Mono 6.12.0 or greater
```

Comment:
This should be

Reply:
That's not accurate though, it's not at all t he minimum required version.

Case in point, we were using 6.12.0.114 just a few weeks ago. And the latest Windows version provided upstream is 6.12.0.107...